### DISCUSSION TITLE
Tailwindcss purges whitelisted tag

### DISCUSSION THREAD

**Discussion by ahoiroman** [tailwindlabs/tailwindcss#2470] [Help] [answered]
I would like to preserve a line like:

```    
h3 {
        @apply text-2xl mt-8 mb-4;
   }
```

Unfortunately, tailwind is purging the h3-tag :-/

This is the config:

```
purge: {
        content: ['./resources/js/**/*.vue',],
        options: {
            whitelist: ['h1', 'h2', 'h3'],
        }
    },
```

**Comment by adamwathan** [comment, marked as answer]
Can you create a GitHub repository that reproduces the issue? What you've posted above works as I'd expect for me.

Also note that Tailwind preserves all base HTML elements by default as of v1.8.0, so you don't need to whitelist things like h3 anymore at all.

**Comment by ahoiroman** [comment]
Hi Adam and thanks for your response. I double-checked `packages.json` and mentioned that I used tailwind `1.7.1`. After updating everything works as expected. Thanks and sorry :-)

**Reply by simonswiss** [reply]
Glad you got it sorted!

And [here's the documentation](https://tailwindcss.com/docs/controlling-file-size#preserving-html-elements) on the `preserveHtmlElements` option, which will prevent base HTML element selectors from being purged, and is turned on by default.
